@Alyx/newExp.m

```diff
@@ -62,26 +62,22 @@
 
 % Main repository is the reference location for which experiments exist
 [expPath, expRef] = dat.expPath(subject, floor(expDate), expSeq, 'main');
-% ensure a unique `expRef` and `expPath` can be made
-assert(~any(file.exists(expPath)), ...
-  sprintf(['Something went wrong as experiment folders already exist for'...
-  '\n"%s" in \n%s or \n%s. \nIf these folders are empty, delete them and '...
```

Comment:
I think something like this should be left in - it's helpful to know what to do in this case

Reply:
> Finally, because this problem usually arises because the folder is created first in the local repo, this won't occur very often in the future because the empty folder is removed in the catch block.

Yes, true. I think having some note about where these folders exist that caused the issue is still helpful because it gives people like trainers a quick thing to check without having them pour through additional documentation

Reply:
I won't leave it in because it very much depends on what the issue is.  The reason we don't just delete the folders automatically is because it requires the experimenter's discretion: if the folder is empty it can be deleted but if there are data in there it might be preferable to copy the folder to the main repo rather than deleting.  This could also conceivably occur because of a paths issue, i.e. the main repo path changes, in which case you wouldn't want to change the local folder but rather change the main repo path in the paths file.